### PR TITLE
fiddle with DConf

### PR DESCRIPTION
**Pull request by aG0aep6G**
* drop old DConf links; they're accessible through the DConf 2015 site
* drop heading
* give some room to breathe

iconscreenshot1

### CODE REVIEW

**Comment by CyberShadow** [comment]
See https://github.com/D-Programming-Language/dlang.org/pull/796

**Comment by aG0aep6G** [comment]
From #796:

> Suggestion: Just put "DConf 2015: Call for submissions open through Feb 27!" Move the other two editions in a submenu or drop entirely.

Are the older links anywhere else? I wouldn't like to drop them completely. I'd rather not make a whole new home for them right now. I just mean to un-uglify the home page a bit.

**Comment by CyberShadow** [comment]
You could put them in the community menu. Also, there are links to DConf 2014/2013 on the DConf 2015 website.

**Comment by andralex** [comment]
yah, please do by @CyberShadow

**Comment by aG0aep6G** [comment]
Updated. See summary.

File: index.dd [hunk 1]
```diff
@@ -4,12 +4,9 @@ $(D_S D Programming Language,
 
 $(SECTION3 The D programming language. $(SPAN style="display: inline-block", Modern convenience. Modeling power. Native efficiency.),
 
-Conference websites, videos and slides:
-$(UL
-$(LI $(LINK2 http://dconf.org/2015, $(B $(RED DConf 2015))))
-$(LI $(LINK2 https://archive.org/search.php?query=title%3A%28DConf%202014%29$(AMP)sort=+publicdate, $(B $(RED DConf 2014))))
-$(LI $(LINK2 http://dconf.org/2013/schedule/index.html, $(B $(RED DConf 2013)))))
-$(BR)
+<br>
```
Comment (CyberShadow) [inline index.dd]: `$(BR)`?

**Comment by CyberShadow** [comment]
> You could put them in the "Community" menu.

?

**Pull request merged**